### DISCUSSION TITLE
CFW的什么问题？

### DISCUSSION THREAD

**Discussion by lshilshi** [SeaHOH/GotoX#156] [一般]
win7 X64 + Chrome X64 v85
Gotox V3.6.1
```
22:53:28 W 2606:4700::6811:b044 _create_ssl_connection 'https://AAA.BBB.wor
kers.dev' 返回 OSError(10065, '[WinError 10065] 套接字操作尝试一个无法连接的主机
。')，重试
```
部署的workers在web上最末页的测试中，输入被墙网址都是显示ok
直接在地址栏中访问，好像一直都是空白加载中
……
类似信息不断出现，不过能正常翻Q
Google正常
1024可以浏览，看不了图片……
 以上。

**Comment by SeaHOH** [comment]
日志中应该不会出现 `_create_ssl_connection 'https://***` 这样的信息。

我对你的问题不太能理解，比如“最末页的测试”、“直接在地址栏中访问”等，能否重新更详尽准确地描述你的问题？

图片无法加载无非几种情况：
- 页面使用懒加载，但 javascript 被禁用
- 图片源站部署了防盗链，但你的浏览器使用了 referer、cookies 等隐私策略
- 源站被墙，同时 GotoX 自动规则无法将它路由到 CFW 规则

观察 GotoX 日志中相关图片的请求记录，即可判断是否为最后一种情况。如是，可手动添加 CFW 规则。

**Comment by lshilshi** [comment]
> 日志中应该不会出现 `_create_ssl_connection 'https://***` 这样的信息。
> 
> 我对你的问题不太能理解，比如“最末页的测试”、“直接在地址栏中访问”等，能否重新更详尽准确地描述你的问题？
> 
> 图片无法加载无非几种情况：
> 
> * 页面使用懒加载，但 javascript 被禁用
> * 图片源站部署了防盗链，但你的浏览器使用了 referer、cookies 等隐私策略
> * 源站被墙，同时 GotoX 自动规则无法将它路由到 CFW 规则
> 
> 观察 GotoX 日志中相关图片的请求记录，即可判断是否为最后一种情况。如是，可手动添加 CFW 规则。

是1024的图片的问题，那看来可能是lazyload的问题了……
后面我再测试测试
现在腰不行了……
